### PR TITLE
Embedded video: model

### PR DESCRIPTION
**Pull request by dasileker**
### Youtube Page (Embedded Video Project For Microverse)

This Embedded video we :

- Created a page that looks like a Youtube page.

- HTML that contained all the forms.

- CSS that gives the page a similar look to Youtube Page.

- Jpegs & embedded video.

- added a README.MD file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
-# Youtube
+# Youtube page 
+> This project consists of building an HTML document that matches the appearance of youtube video player page.
+
```
Comment (elmejdki) [inline README.md:7]: Please add a screenshot of your project

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+[Live Demo Link](https://rawcdn.githack.com/dasileker/Youtube/f4758154d538e17becf46387252db440d5e24384/youtube.html)
+
+
+
+**Author 1**
+GitHub [@dasileker](https://github.com/dasileker)
+
+**Author 2**
+GitHub [@olyadi](https://github.com/CornerstoneII)
```
Comment (elmejdki) [inline README.md]: Add other social media profile links like twitter and linkeding

File: youtube.html [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+      </div>
+      <section class="caption bg-white p-1 mt-half">
+        <h2 class="caption-text">
+          10 Best Places to Visit in Morocco - Travel Video
+        </h2>
+        <div class="channel-body">
+          <img alt="maroc" class="channel-img pb-1" src="assets/maroc.jpg" />
+          =======
+          <img alt="maroc" class="channel-img pb-1" src="assets/maroc.jpg" />
+          >>>>>>> master
```
Comment (elmejdki) [inline youtube.html]: Please remove these unnecessary lines

File: youtube.html [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+                <img
+                  class="bg-youtube"
+                  src="assets/youtube-logo.png"
+                  alt="YouTube"
+                />
+                <img
+                  class="bg-youtube"
+                  src="assets/youtube-logo.png"
+                  alt="YouTube"
+                />
```
Comment (elmejdki) [inline youtube.html]: remove one image button and leave only one

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## Great job guys :+1: :1st_place_medal: :clap:  you have made the best youtube clone I have ever seen, Great use of semantic HTML, great design it looks exactly like the original :100: , but you still have some changes :hammer: :wrench: :toolbox: to make in order to get your project approved :checkered_flag: 

- [x] Next time keep your commit messages short and meaningful.

### Please make all the required changes before you submit another code review.
## Happy Coding :tophat:

File: README.md (hunk 1 above)
Comment (dasileker) [reply README.md:7]: done

File: README.md (hunk 2 above)
Comment (dasileker) [reply README.md]: done

File: youtube.html (hunk 3 above)
Comment (dasileker) [reply youtube.html]: thanks and done

File: youtube.html (hunk 4 above)
Comment (dasileker) [reply youtube.html]: thanks and done

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle:  
## Hi, guys it's me again thank you for making the required changes, but you still need to make one more change.

- [x] Please update your live demo link it's not working

### Please make the required changes before you submit another code review.
## Happy coding :tophat:

**Comment by dasileker** [comment]
I corrected the problem

**Review by kitpao — APPROVED** [review]
# 👋 Hello Team,

I see you have complied with my colleague's corrections and your page is looking awesome! great job with this project 🥇

### 👏 Your project has been approved 👏
### 🚀 You may merge to master and submit the project completition form 🚀

___
##### 👉 If you have any doubts you can contact me in slack as @kitpao

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

**Pull request merged**